common-pg/src/main/resources/com/socrata/pg/store/schema/20221116-create-rollup-table-name-seq.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <changeSet author="Paul Maksimovich" id="20221116-create-rollup-table-name-seq">
+        <sql>
+            create sequence rollup_table_name_seq
+                minvalue 0
+                cycle;
```

Comment:
It will be quite the accomplishment if we ever flow a bigint here!

Reply:
But if and when it does I want it!